src/components/customRenderers/CreatableSelect/CreatableSelect.tsx

```diff
@@ -5,6 +5,33 @@ import { CreatableSelect as Creatable } from '@amsterdam/bmi-component-library';
 import { ControlProps } from '@jsonforms/core';
 import useCustomControl from '../../../hooks/useCustomControl';
 
+type SchemaOptionType = {
+	const: string;
+	title: string;
+};
+
+type SelectOptionType = {
+	value: string;
+	label: string;
+};
+
+function getOptionForValue(options: SchemaOptionType[], value: string): SelectOptionType | undefined {
+	const option = options.find((option) => option.const === value);
+	return option
+		? {
+				value: option.const,
+				label: option.title,
+				// eslint-disable-next-line no-mixed-spaces-and-tabs
+		  }
+		: value
```

Comment:
Shouldn't this be in parenthesis? I'm guessing this works, so it probably is just a styling issue.

Reply:
Prettier decides..